### DISCUSSION TITLE
sqlalchemy-oauth example in the repo seems to be broken?

### DISCUSSION THREAD

**Discussion by Sharjeel50** [fastapi-users/fastapi-users#1251] [General]
Just cloned the repo and ran the[ sqlalchemy-oauth ](https://github.com/fastapi-users/fastapi-users/tree/master/examples/sqlalchemy-oauth/app) example but I'm getting the following error - 

`pydantic_core._pydantic_core.PydanticSerializationError: Unable to serialize unknown type: UUID('29214366-5392-471f-b4ec-06386fcf1dfa')`

Is there a quick fix for this?

thanks

**Comment by frankie567** [comment]
I've tried the examples and they seem to work well.

The error you get is related to Pydantic V2. Can you check that your version of `fastapi-users` is at least `12.1.0`?

**Comment by Sharjeel50** [comment]
hey @frankie567, thanks for getting back to me so quickly, it looks like my Python version (`3.7.16` & `3.9`) may have been the issue, all `fastapi-users` installations were below `12.1.0` when checking site-packages

I've now updated my Python version to `3.11`and retested it, everything looks to be working well as mentioned.

Thanks again.
